improved readme (#8)
zkVerify/compose-vflow-simplified: README.md

        ```
        cd /var/lib/docker/volumes/vflow_node-data/_data/node/zkv_relay/chains/zkv_mainnet
        ```
7. Note the owner and permissions of the existing `db` directory, then delete it.
8. Extract the downloaded zkVerify snapshot and move its `db` directory into the current directory.
9. Ensure the new `db` directory has the same permissions as the original db directory.
10. Return to the project directory and start the node:
   ```shell
   ./scripts/start.sh
   ```

> 11. Verify the snapshot is working by checking the node's Docker logs to ensure the **relay** and **para** chains’ block heights start near their respective current heights and continue steadily increasing.

Reviewer: ```suggestion
11. Verify the snapshot is working by checking the node's Docker logs to ensure the block height starts near its respective current chain height and continue steadily increasing.
```
